support/cas-server-support-rest-x509/build.gradle

```diff
@@ -2,6 +2,7 @@ description = "Apereo CAS X509 REST Implementation"
 dependencies {
     implementation project(":support:cas-server-support-rest")
     implementation project(":support:cas-server-support-x509-core")
+    implementation project(":support:cas-server-support-x509-webflow")
```

Comment:
Then the X509CredentialExtractor would need to move to another module.  Suggestions?

Reply:
I must admit I don't follow. What prompted or required you to include the webflow module in here? What did or does it provide for this work that you otherwise couldn't access?